lms/static/sass/views/_login-register.scss

```diff
@@ -358,10 +360,6 @@ $sm-btn-linkedin: #0077b5;
         font-size: 0.8em;
         border-color: $lightGrey1;
 
-        &:nth-of-type(odd) {
-            margin-right: 13px;
-        }
-
         .icon {
             color: white;
```

Comment:
Please use a [Sass variable](https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Fbase%2F_variables.scss#L66) for this color.